### PR TITLE
Update docs with more details on process of staking + joining a DAO.

### PR DESCRIPTION
**Pull request by ezekiiel**
This updates the DAO quickstart process to include a step about "joining the DAO" which guides folks through joining the DAO themselves and getting their friends to stake before creating the first proposal.

### CODE REVIEW

File: docs/quickstart/joining.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 2
+---
+
+# Joining your DAO
+
+To join a DAO DAO DAO you must "register to vote" by staking your
```
Comment (elsehow) [inline docs/quickstart/joining.md:7]: nice - I am a fan of this metaphor, as you know.

File: docs/quickstart/joining.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+governance tokens with the DAO. Once you are registered to vote you
+can vote on all future proposals, but you cant go back and vote on
+proposals that were created before you registered.
+
+To register to vote visit your DAO's page on DAO DAO and look in the
```
Comment (elsehow) [inline docs/quickstart/joining.md:12]:
```
To register to vote, visit...
```

File: docs/quickstart/joining.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+right sidebar:
+
+![](/img/quickstart/vote-staking.png)
+
+From here you can press the "Stake tokens" button and a modal will
+appear asking you how many you'd like to stake:
```
Comment (elsehow) [inline docs/quickstart/joining.md:18]: ```how many tokens you'd like to stake```

File: docs/quickstart/joining.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+![](/img/quickstart/vote-staking-50.png)
+
+By staking your tokens you are registering those tokens for
```
Comment (elsehow) [inline docs/quickstart/joining.md:22]:
```
By staking your tokens, you are registering those tokens for voting.
```

File: docs/quickstart/joining.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+voting. Any tokens you do not stake won't count towards your voting
+power. If you want your votes to matter as much as possible, you'll
+likely want to stake all of them. :)
+
+There are some rare situations where you may not want to stake all of
+your tokens. For example:
+
+1. Some DAOs require that some tokens are deposited to create a
+   proposal. If you plan on making a proposal you may leave one
```
Comment (elsehow) [inline docs/quickstart/joining.md:31]: `If you plan on making a proposal, you may leave one deposit's worth of tokens unstaked`

File: docs/quickstart/joining.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+   deposit worth of tokens unstaked.
+2. Some DAOs have an unstaking duration which is the amount of time
```
Comment (elsehow) [inline docs/quickstart/joining.md:33]: `Some DAOs have an unstaking duration, defined as the amount of time you must wait after unstaking your tokens before those tokens are available to you. If you are holding tokens in a DAO with a non-zero staking duration, and you think you may want to sell or trade those tokens quickly in the future, you may consider leaving some tokens unstaked.`

**Review by elsehow — CHANGES_REQUESTED** [review]
Fantastic work here - love the new additions, the diagram, and many thanks for the linting cleanups and increased screenshot resolution (can't wait until screenshots are automated on build!).

Left a few, mostly minor comments in my review. Have a look and let me know if you have any questions!

**Comment by ezekiiel** [comment]
Thanks for the review. Addressed the comments with the force push. :)

**Pull request merged**